frontend/lib/norent/site.tsx

```diff
@@ -25,6 +25,7 @@ import { NorentAboutPage } from "./about";
 import { NorentAboutYourLetterPage } from "./the-letter";
 import { NorentLogo } from "./components/logo";
 import { NorentLetterBuilderRoutes } from "./letter-builder/steps";
+import { NorentLogOutPage } from "./log-out";
```

Comment:
Err do you mind calling the file `logout`, just cuz that's how I spell it for all the tenant platform-related logout stuff... 😣